src/Pos/Util/UserSecret.hs

```diff
@@ -81,34 +99,70 @@ instance Bi UserSecret where
         keys <- get
         return $ def & usVss .~ vss & usPrimKey .~ pkey & usKeys .~ keys
 
+#ifdef POSIX
+-- | Constant that defines file mode 600.
```

Comment:
It'd be nice if there was a comment saying what mode `600` corresponds to (afaik, “only owner can read and write”)